dl1_data_handler/transforms.py

```diff
@@ -153,12 +153,21 @@ def __call__(self, example):
         array = []
         for i, (val, des) in enumerate(zip(example, self.description)):
             if des['base_name'] == 'image':
-                image = val
+                image = val.T
```

Comment:
why?

Reply:
because the reader returns the images with shape (num_pixels, channels)